### ISSUE TITLE
Question about batch size and training time in ACTIVE-PC

### ISSUE THREAD

**Issue by yunandyan** [wangzy01/ACTIVE-Action-from-Robotic-View#5]
Hi authors, thanks for the great work and for releasing the ACTIVE dataset and code.

I have a question regarding the training details of ACTIVE-PC. In the paper, the optimizer, learning rate schedule, number of epochs (50), and GPU (RTX 4090) are specified, but I could not find information about:

- The batch size used for training
- The approximate training time

These details would be very helpful for reproducing the results and for future extensions based on your method.

Thanks in advance!
